collectors/ioping.plugin/README.md

```diff
@@ -1,86 +1,120 @@
 <!--
 title: "ioping.plugin"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/ioping.plugin/README.md
+description: "Netdata's ioping plugin allows you to measure your disk latency."
 -->
 
 # ioping.plugin
 
-The ioping plugin supports monitoring latency for any number of directories/files/devices,
-by pinging them with `ioping`.
+The ioping plugin supports monitoring latency for any number of directories/files/devices, by pinging them with `ioping`.
+When your disks don't respond in the time you defined in `ioping.conf`, the plugin raises an alert (`ioping_disk_latency`).
 
-A recent version of `ioping` is required (one that supports option `-N`).
-The supplied plugin can install it, by running:
+The collector is enabled if Netdata detects that you have `ioping` installed. Otherwise, it is disabled. 
 
-```sh
-/usr/libexec/netdata/plugins.d/ioping.plugin install
-```
+## Prerequisites
+
+To use this plugin, you need to install `ioping`.
+
+## Installing ioping
+
+You can install `ioping` using the plugin.
+The following command will download, build and install the correct `ioping` version as `/usr/libexec/netdata/plugins.d/ioping`.
+
+To install `ioping`, run:
+
+   ```sh
+   /usr/libexec/netdata/plugins.d/ioping.plugin install
+   ```
+   Once `ioping` is installed, Netdata will automatically start collecting disk latency data.
```

Comment:
After installing we need:

- configure "destination".
- restart netdata service.

Reply:
Configuration is covered in the next sections, but let @DShreve2 know if there is info missing.

Reply:
I answered to

> Line 28: Do I need to restart Netdata so the change comes into effect?